aptos-move/aptos-release-builder/data/example.yaml

```diff
@@ -0,0 +1,472 @@
+---
```

Comment:
This is an example release config that contains all the information we could set using on chain config. Governance proposals could be generated automatically from this yaml file.